### ISSUE TITLE
Using vispy Mesh in napari viewer

### ISSUE THREAD

**Issue by cakeinspace** [napari/napari#3334]
In vispy we can display the mesh and the wireframe associated with it. Is this functionality in napari as well. So far I only found a mesh display in the form of a view_surface function. Alternatviely can we display the wireframe of a mesh in napari or use vispy classes for display in the napari viewer.

**Comment by liaprins-czi** [comment]
### What level of design is needed? (Choose all that apply)
- [X] **User experience research:**  high-level recommendation/exploration of user needs, design heuristics, and / or best practices to inform a design experience  
(Use this option when you feel there’s a challenge to be solved, but you’re curious about what the experience should be — may involve research studies to understand challenges/opportunities for design)
- [X] **Information flow / conceptual:** organizing and structuring of information flow and content, including layout on screen or across multiple steps
- [X] **Visual:** creating mockups, icons, etc
(If choosing this level alone, it means that the content to be mocked up and its organization is already known and specified)

### Is design a blocker?
- [X] **Yes:** engineering cannot proceed without a design first
- [ ] **No:** engineering can create a first version, and design can come in later to iterate and refine

If selecting **Yes**, how much design input is needed to unblock engineering? For example, is a full, final visual design needed, or just a recommendation of which conceptual direction to go?
Depends if there are existing patterns we can identify to start with, and how much designing will need to go into refining them for our specific use cases.

### T-shirt size
- [ ] ●○○○○ XS: able to be done in a typical workday or two
- [ ] ●●○○○ S: able to do within one sprint, if also working on other things in the sprint
- [ ] ●●●○○ M: able to do within one sprint, if it is the sole focus
- [X] ●●●●○ L: will likely take more than one sprint to complete, but less than two
- [ ] ●●●●● XL: a very large experience to be determined or overhauled, will need to be broken down further

**Comment by aloofhawk** [comment]
i think i added the wrong label its more of a extending current feature question ill close it and open a new issue

**Comment by alisterburt** [comment]
Hi @aloofhawk!

@kevinyamauchi and I discussed this a bit in #2970 and I'm working on it right now. #3475 is a stepping stone which brings us in line with versions of vispy which have this feature so it shouldn't be hard now 😄 

It's definitely possible to add vispy meshes to the canvas but I don't think we would want to encourage that sort of solution

edit: sneak peek
![image](https://user-images.githubusercontent.com/7307488/137196801-c1941d43-f669-48c0-99e9-ea558886b852.png)
